src/pages/Detail/Comment.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import S from './Styled.Detail';
+
+const Comment = ({ id, nickname, comment, created_at, getData }) => {
+  const [reply, setReply] = useState('');
+  const [replyList, setReplyList] = useState([]);
+  const [replyOpen, setreplyOpen] = useState(false);
+
+  const replyInput = e => {
+    setReply(e.target.value);
+  };
+
+  const toggleReply = () => {
+    setreplyOpen(replyOpen => !replyOpen);
+  };
+
+  const today = new Date();
+  const year = today.getFullYear();
+  const month = today.getMonth() + 1;
+  const day = today.getDate();
+  const date = `${year}.${month >= 10 ? month : '0' + month}.${
+    day >= 10 ? day : '0' + day
+  }`;
+  const time = today.toLocaleTimeString([], {
+    hourCycle: 'h23',
+    hour: '2-digit',
+    minute: '2-digit',
+  });
+
+  const addComment = e => {
+    e.preventDefault();
+    if (reply.trim().length === 0) return;
+
+    const newComment = {
+      nickname: '싱그러운 새싹',
+      parent_comment_id: id,
+      comment: reply,
+      created_at: `${date} ${time}`,
+    };
+
+    setReplyList(prevList => {
+      return [...prevList, newComment];
+    });
```

Comment:
함수에 다른 로직 없이 바로 return된다면 중괄호와 return은 생략해도 될 것 같습니다.
```suggestion
    setReplyList(prevList => [...prevList, newComment]);
```